internal/services/applications/application_resource_test.go

```diff
@@ -1572,3 +1594,20 @@ resource "azuread_application" "test" {
 }
 `, data.RandomInteger)
 }
+
+func (r ApplicationResource) password(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azuread" {}
+
+data "azuread_client_config" "current" {}
+
+resource "azuread_application" "test" {
+  display_name = "acctest-APP-%[1]d"
+  owners = [ data.azuread_client_config.current.object_id ]
+
+  password {
+    display_name = "acctest-APP-%[1]d"
```

Comment:
(display name change as suggested above)

```suggestion
    display_name = "acctest-appPassword-%[1]d"
```